package.json

```diff
@@ -79,7 +80,7 @@
     "build:upload": "browserify frontend/src/upload.js -g uglifyify -o public/upload.js",
     "build:download": "browserify frontend/src/download.js -g uglifyify -o public/download.js",
     "build:version": "node scripts/version",
-    "build:l10n": "cp node_modules/l20n/dist/web/l20n.min.js public",
+    "build:3rd": "cp node_modules/l20n/dist/web/l20n.min.js public && cp node_modules/babel-polyfill/dist/polyfill.min.js public",
```

Comment:
I'll give you a 💵 if we rename this to something more grep-able like "build:vendor".

You could even combine the two:

```sh
cp node_modules/l20n/dist/web/l20n.min.js node_modules/babel-polyfill/dist/polyfill.min.js public
```